vllm/core/block/common.py

```diff
@@ -30,36 +30,32 @@ class RefCounter(RefCounterProtocol):
             to initialize the reference counter with.
```

Comment:
update docstring

Reply:
fixed in [07f01fe](https://github.com/vllm-project/vllm/pull/5968/commits/07f01fed1dcb31fcbd1c7cfaf95a504fd8f23694)